### ISSUE TITLE
Colony.dat deleted by server crash, now can't delete manually.

### ISSUE THREAD

**Issue by Xendisabu** [ldtteam/minecolonies#5248]
### Minecolonies version # Update to the newest minecolonies and structurize alpha first and see if your issue might be already fixed!

Version: 0.11.909-ALPHA

### Expected behavior

Be able to delete the colony and then place a new colony in a similar location.

- 
One of the colonies on our multiplayer server seems to have become corrupted during a server crash (unclear if it was the cause of the crash). The corresponding colony3_0.dat file was deleted during the crash, but some part of the game thinks it still exists, even though the file is gone.

### Actual behaviour

Can not delete colony 3, colony 3 does not appear in the /mc colony list, but when a town hall is placed, the error received says that colony 3 is too close.

- 
Minecolonies claims there is no colony with the specified ID (3) on the server when a delete command is used, however when the player tries to place a townhall, it will not let the player build it saying colony 3 is too close.

### Steps to reproduce the problem

Reproduction steps are unknown. Several players were on the server at the time of the crash, two were supplying builder's huts with materials.

### Logs: Latest.log and crashlog at https://gist.github.com/

- latest.log: https://gist.github.com/Xendisabu/5795bc87c8336a362eadd4b80d223928
- crashlog: Crash log was empty.

**Comment by Asherslab** [comment]
use the backups that minecolonies makes to restore the colony_3's dat and see if you can delete it after that

**Comment by Raycoms** [comment]
Its because the claim area is still there. use /mc colony claim with "false" to unclaim the area.

**Comment by Xendisabu** [comment]
I have tried that, the server responds with "Colony for id 3 could not be found in this dimension." I also looked in the server files and there are no Colony 3 files. If I run the "load backup" command, I get a success message, but if I try to delete the colony, I get the same message as above that the colony is not found.

**Comment by Raycoms** [comment]
no colony3 files in the backup zips either?

**Comment by someaddons** [comment]
it there is no colony 3 in the /mc colony list then there is no colony 3. What you're experiencing is a chunk which is still claimed for the no longer existing colony. Use the claim commands to unclaim colony 3 in like a 30 chunk radius from where you try to place the colony and it'll work

**Comment by Xendisabu** [comment]
@Raycoms Correct, colony 3 is not present in the backup zips nor in the colony list.
@someaddons the max chunk radius to claim is 10, and even when repeating the command from different areas within the old colony's radius it still claims that the colony does not exist.

**Comment by Raycoms** [comment]
can you just claim the area to an existing
 one, and then unclaim again?

**Comment by Xendisabu** [comment]
It allows me to claim/unclaim for other colonies, however when I try to place the town hall, it places but says "Colony with ID 3 is nearby" and "there is an existing colony too close." The two buttons for Create New Colony and Delete own colony are grayed out.

**Comment by Raycoms** [comment]
Hmm, due to the corruption the best way probably is to create a new colony outside of this zone. This will then get the colony id 3 and then you should be able to properly unclaim those chunks.

**Comment by Xendisabu** [comment]
To resolve the issue, I claimed the entire area of the erased colony 3 for colony 2, then unclaimed it. At this point, I was still unable to place colony 3 in the original area. I then created a colony in a different location which took on the name of colony 3. After the first colonist showed up, I deleted the colony. I returned to the original location of colony 3 and tried placing the town hall. It still had the warning that colony 3 was too close, but allowed me to create the new colony anyway. After creating the colony, I checked and the list shows the new colony as number 3. At this point it appears to be working.
